cmd/registry/cmd/annotate/annotate.go

```diff
@@ -81,9 +81,9 @@ func Command() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&filter, "filter", "", "Filter selected resources")
-	cmd.Flags().BoolVar(&overwrite, "overwrite", false, "Overwrite existing annotations")
-	cmd.Flags().IntVar(&jobs, "jobs", 10, "Number of actions to perform concurrently")
+	cmd.Flags().StringVar(&filter, "filter", "", "filter selected resources")
+	cmd.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite existing annotations")
+	cmd.Flags().IntVar(&jobs, "jobs", 10, "number of actions to perform concurrently")
```

Comment:
We have various descriptions for `--jobs` that perhaps we should consolidate? Also, some commands include a `-j` short option and others don't.

Reply:
Agreed: `jobs`, `filter`, `force` should probably all have consistent descriptions, etc.

Reply:
I've added a check that verifies that those three commands have consistent usage and shorthand options.